core/templates/dev/head/base.html

```diff
@@ -122,7 +122,7 @@
     {% if iframed %}
       {{ warnings_and_loader() }}
     {% else %}
-      <div class="oppia-base-container" ng-class="{'oppia-sidebar-menu-open': sidebarIsShown, 'oppia-sidebar-menu-closed': !sidebarIsShown}">
+      <div class="oppia-base-container" ng-class="{'oppia-sidebar-menu-open': sidebarIsShown, 'oppia-sidebar-menu-closed': !sidebarIsShown}" ng-swipe-left="openSidebar()" ng-swipe-right="openSidebar()">
```

Comment:
1) I tested by connecting my android device via USB, and start a remote debugging session on my phone. I swiped left and right on my phone and sidebar closed and opened.
2) I did not think this fully through. No the sidebar shouldn't close on swiping right neither should it open on swiping left. I will correct this in the next commit. Sorry for carelessness on my part.
3) Hmm...I haven't tested this yet. I have to learn how to simulate swipes on my desktop because I do not have a widescreen device. I will learn how to test this and address this in the next commit :)